modules/administration_manual/pages/configuration/server/occ_command.adoc

```diff
@@ -2663,18 +2661,20 @@ will exit with 1. Only applicable on get.
 | `--error-if-not-exists`           | Checks whether the setting exists before deleting it.
 |====
 
-Here's an example of how you would set the email address of the user `layla`.
+[IMPORTANT:] In case you want to change the email address, do not use `user:settings` but the `user:modify` command.
```

Comment:
I'd end this sentence as: "use the `user:modify` command, not the `user:settings` command." This approach should require less cognitive load on the user.

Reply:
fixed